src/components/Blogs/Blogs.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SectionHeading from '../SectionHeading/SectionHeading';
+import style from './Blogs.module.css';
+import CoreLinkBtn from '../CoreLinkBtn/CoreLinkBtn';
+import PostCard from '../PostCard/PostCard';
+import { useState } from 'react';
+import blogsData from './blogsData';
+
+const ThreeBlogs = blogsData.slice(0, 3);
+const Blogs = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const toggleExpand = () => {
+    setIsExpanded(!isExpanded);
+  };
+  return (
+    <div className={style.blogs}>
```

Comment:
I don't want to put much pressure on you, but in the future use "Box" from MUI instead of div